frontend/lib/util/format.ts

```diff
@@ -66,25 +66,43 @@ function isYesterday(date: Date): boolean {
   );
 }
 
-export function formatDateTime(date: Date) {
-  const today = new Date();
+export function formatDateTimeFull(date: Date) {
```

Comment:
Can you add a simple unit test for this function?

Reply:
Added some unit tests in [b0e0d9f](https://github.com/kiesraad/abacus/pull/1154/commits/b0e0d9fa7ad4b5ac44992f14699b0d5298c00d0e)